compiler/env/OMRClassEnv.hpp

```diff
@@ -141,6 +143,14 @@ class OMR_EXTENSIBLE ClassEnv
     */
    const TR::TypeLayout* enumerateFields(TR::Region& region, TR_OpaqueClassBlock * clazz, TR::Compilation *comp) { return NULL; }
 
+   /**
```

Comment:
For doxygen format you can look at https://github.com/eclipse/omr/blob/master/compiler/p/env/OMRCPU.hpp as an example